lib/taggable.rb

```diff
@@ -272,7 +272,7 @@ def tag
 
   # Index all the filters for pulling works
   def filter_ids
-    filters.pluck :id
+    (tags.pluck(:id) + filters.pluck(:id)).uniq
```

Comment:
This slightly alters the way that searching for non-canonical tags works (it used to just fall back to searching the :tag field for matching strings), but I actually much prefer this way of handling it.